examples/example-ui-react/src/App.tsx

```diff
@@ -1,10 +1,17 @@
 import * as React from 'react'
 import '../../../tokens/css/variables.css'
 import LeoButton from '../../../react/button'
+import Tooltip from '../../../react/tooltip'
 import styles from './App.module.css'
 import Input from '../../../react/input'
 import Dropdown from '../../../react/dropdown'
 
+import { color } from '../../../tokens/css'
+
+color.white
+
+let f = '#FFFFFF'
+
```

Comment:
Is this for testing for errors? Although I'm a bit confused what `f` is used for 😄

Reply:
Whoops, I was probably using it to see what colors are available :flushed: